### DISCUSSION TITLE
Add a metadata to fake players so external plugins can detect them

### DISCUSSION THREAD

**Discussion by SkytAsul** [BG-Software-LLC/WildLoaders#81] [Pending Feature Requests]
Because your chunk holders are basically fake players, sometimes those players instance can leak out and reach external plugins. In those cases, it can cause issue.
For instance, my plugin BeautyQuests sometimes tries to fetch data about those not-existing players, thus crashing.
An easy fix would be to add an "NPC" metadata to the entity, through the `Entity#setMetadata` method.

**Comment by OmerBenGera** [comment]
Bukkit#getPlayers shouldn't return the NPCs, do you use NMS by any chance to get the players list?

Anyways, open an issue so I can fix it asap (I am not at home right now, so keeping it here will just cause it to be ignored)
